src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.quarto.component.spec.ts

```diff
@@ -129,6 +129,7 @@ export async function prepareStartedGameFor<T extends AbstractGameComponent>(
     }
     const update: Partial<Part> = {
         playerOne: UserMocks.OPPONENT_MINIMAL_USER,
+        playerOneElo: 0,
```

Comment:
c't'une update, no need de spécifier tous les champs, non ?

Reply:
euh, out of date icitte